### PR TITLE
modbus: Mute build warning

### PR DESCRIPTION
**Pull request by kumajaya**

### CODE REVIEW

File: src/com/modbus/modbuslayer.cpp [hunk 1]
```diff
@@ -171,7 +171,7 @@ EComResponse CModbusComLayer::processInterrupt(){
     switch (mConnectionState){
       case e_Connected: {
         CIEC_ANY **apoRDs = mFb->getRDs();
-        unsigned int nrRDs = mFb->getNumRD();
+        unsigned int nrRDs = static_cast<unsigned int>(mFb->getNumRD());
```
Comment (azoitl) [inline src/com/modbus/modbuslayer.cpp]: The correcter and in our current code style solution would be to change the data type of `nrRDs`, `dataIndex`, and `i` to `size_t`.

Comment (kumajaya) [reply src/com/modbus/modbuslayer.cpp]: I just though the rest of the code is not capable to handle size_t.

Comment (azoitl) [reply src/com/modbus/modbuslayer.cpp]: what I see the code would be correcter with size_t but maybe a bit more changes would be needed.

Comment (kumajaya) [reply src/com/modbus/modbuslayer.cpp]: > what I see the code would be correcter with size_t but maybe a bit more changes would be needed.

I just mute the warning for now.

**Review by azoitl — APPROVED** [review]

**Pull request merged**